### PR TITLE
Add Verify Job to check consistency between generate yaml and the yam…

### PR DESCRIPTION
**Pull request by asifdxtreme**
…l in the repo

https://github.com/volcano-sh/volcano/issues/347

### CODE REVIEW

File: hack/check-generated-yaml.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+if ! diff ${VK_ROOT}/installer/volcano-development.yaml ${RELEASE_FOLDER}/volcano-latest.yaml ; then
+	{
+		echo
+		echo "The Generated yaml is different from the one in installer/volcano-development.yaml"
+		echo "Suggest to sync both the files"
+		echo
+	} >&2
+	false
+fi
```
Comment (TommyLike) [inline hack/check-generated-yaml.sh]: Missing newline here

Comment (asifdxtreme) [reply hack/check-generated-yaml.sh]: updated

**Pull request merged**